### PR TITLE
feat: [PROD-10462] add generic slider component for number input

### PR DESCRIPTION
**Pull request by esasova**
Before creating a PR, please check that:
- [ ] Code is clean
- [ ] Tests are still working (cypress tests and `yarn test`)
- [ ] Changes don't cause new errors or warnings in browser console
- [ ] Documentation is up-to-date
- [ ] Breaking changes are clearly identified with [conventional commits](https://kapeli.com/cheat_sheets/Conventional_Commits.docset/Contents/Resources/Documents/index)

### CODE REVIEW

File: src/components/ScenarioParameters/components/ScenarioParametersInputs/GenericSliderInput.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  }
+  return parameterData.minValue;
+}
+
+function getMaxValue(parameterData) {
+  if (parameterData.maxValue == null) {
+    return DEFAULT_MAX_VALUE;
+  }
+  return parameterData.maxValue;
+}
```
Comment (csm-thu) [inline src/components/ScenarioParameters/components/ScenarioParametersInputs/GenericSliderInput.js:24]: Minor suggestion improvement (even though these lines may be copied from another file :slightly_smiling_face: ), we could rewrite these lines to something like:
```
const getMinValue = (parameterData) => parameterData.minValue ?? DEFAULT_MIN_VALUE;
const getMaxValue = (parameterData) => parameterData.maxValue ?? DEFAULT_MAX_VALUE;
```

File: src/components/ScenarioParameters/components/ScenarioParametersInputs/GenericSliderInput.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  };
+
+  let value = parametersState[parameterData.id];
+  if (value == null) {
+    value = NaN;
+  }
+  return (
+    <BasicSliderInput
+      key={parameterData.id}
+      data-cy={parameterData.dataCy}
```
Comment (csm-thu) [inline src/components/ScenarioParameters/components/ScenarioParametersInputs/GenericSliderInput.js:44]: Note: `parameterData.dataCy` will no longer be used after merging the multi-workspaces branch, the `data-cy` tag value will be generated based on the parameter id and the input type (i.e. "slider" type)

**Review by csm-thu — APPROVED** [review]
LGTM :+1:

**Review by csm-thu — APPROVED** [review]
LGTM :+1:

**Review by nborde-CSM — APPROVED** [review]
LGTM :+1:

**Pull request merged**